spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -64,4 +64,7 @@ private SparkSQLProperties() {}
 
   // Overrides the delete planning mode
   public static final String DELETE_PLANNING_MODE = "spark.sql.iceberg.delete-planning-mode";
+
+  // Overrides the advisory partition size
+  public static final String ADVISORY_PARTITION_SIZE = "spark.sql.iceberg.advisory-partition-size";
```

Comment:
Question: Do we need the write prefix or make it part of the name? It only affects the final write.

Reply:
Shouldn't this be a `write.spark.advisory-partition-size` table property? I wouldn't want to set this in the Spark context.

Reply:
Nevermind, I see this is a Spark default that is overridden by the table property.

Reply:
Correct, there is a table property, a SQL config and a write option.